pkg/cmd/provisioning/ibmcloud/delete_service_id_test.go

```diff
@@ -0,0 +1,213 @@
+package ibmcloud
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/google/uuid"
+	"github.com/spf13/cobra"
+	"github.com/stretchr/testify/require"
+
+	"github.com/IBM/go-sdk-core/v5/core"
+	"github.com/IBM/platform-services-go-sdk/iamidentityv1"
+
+	mockibmcloud "github.com/openshift/cloud-credential-operator/pkg/ibmcloud/mock"
+)
+
+func Test_deleteServiceIDCmd(t *testing.T) {
+	type args struct {
+		cmd  *cobra.Command
+		args []string
+	}
+	tests := []struct {
+		name    string
+		args    args
+		wantErr bool
+	}{
+		{
+			name:    "CreateSharedSecretsCmd with unset API key environment variable should fail",
+			wantErr: true,
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			if err := deleteServiceIDCmd(tt.args.cmd, tt.args.args); (err != nil) != tt.wantErr {
+				t.Errorf("deleteServiceIDCmd() error = %v, wantErr %v", err, tt.wantErr)
+			}
```

Comment:
can we reflow this logic so that the asserts kind of self-document what is happening/expected?
```suggestion
			err := deleteServiceIDCmd(tt.args.cmd, tt.args.args)
			if tt.wantErr {
			    assert.Error(err, ...)
			} else {
			    assert.NoError(err, ...)
			}
```

Reply:
agreed, this is more readable

Reply:
All credit goes to goland autogenerate feature :) I understand the expectation, I will enhance the code accordingly.